README.rst

```diff
@@ -1086,3 +1086,83 @@ And more...
 -----------
 
 Additional code samples are available in the `test directory <https://github.com/irods/python-irodsclient/tree/master/irods/test>`_
+
+
+=======
+Testing
+=======
+
+Setting up and runnning tests
+-----------------------------
+
+The Python iRODS Client comes with its own suite of tests.  Some amount of setting up may be necessary first:
+
+    1. Use :code:iinit to specify the iRODS client environment.
+       For best results, point the client at a server running on the local host.
+
+    2. Install the python-irodsclient along with the :code:`unittest unittest_xml_reporting` module or the older :code:`xmlrunner` equivalent.
+          - for PRC versions 1.1.1 and later: :code:`pip install ./path-to-python-irodsclient-repo[tests]`  (when using a local Git repo)
+                                          or: :code:`pip install python-irods[tests]`  (when installing directly from PyPI)
+          - earlier releases (<= 1.1.0) will install the outmoded :code:`xmlrunner` module automatically
+
+    3. Follow further instructions in the `test README <https://github.com/irods/python-irodsclient/tree/master/irods/test/README.rst>`_
+
+
+Testing S3 parallel transfer
+----------------------------
+
+System requirements::
+
+- Ubuntu 18 user with Docker installed.
+- Local instance of iRODS server running.
+- Logged in sudo privileges.
+
+Run a MINIO service::
+
+  $ docker run -d -p 9000:9000 -p 9001:9001 minio/minio server /data --console-address ":9001" 
+
+Set up a bucket :code:`s3://irods` under MINIO::
+
+  $ pip install awscli
+
+  $ aws configure 
+  AWS Access Key ID [None]: minioadmin
+  AWS Secret Access Key [None]: minioadmin
+  Default region name [None]: 
+  Default output format [None]: 
+
+  $ aws --endpoint-url http://127.1:9000 s3 mb s3://irods
```

Comment:
127.0.0.1 please

Reply:
check.  This was pasted straight from my command history, usually a bad idea : )